buildSrc/src/main/groovy/io.deephaven.java-repository-conventions.gradle

```diff
@@ -1,6 +1,12 @@
 repositories {
     mavenCentral()
-
+    // Uncomment the following if doing local development work that depends on SNAPSHOTs
```

Comment:
probably should remove this? or if helpful generally, guard it behind a -P flag, and add mavenLocal for any locally installed builds?

personally i'd prefer it not be there, since it could have surprising other effects, but i can see either way

Reply:
I don't like the idea of `-P` guards in buildSrc. Maybe we've already done that elsewhere? That said, I think there are some development workflows we may want to make easier now that jpy is external (and deephaven-csv). I'll remove for now.